docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -128,21 +133,21 @@ read-write by a single Kubernetes node.
 by AzureDisk.)
 ``storage: 20Gi`` means the volume has a size of 20
 `gibibytes <https://en.wikipedia.org/wiki/Gibibyte>`_.
-(You can change that if you like.)
 
-Create ``mongoclaim`` in your Kubernetes cluster:
+You may want to update the ``spec.resources.requests.storage`` field in both
+the files to specify a different disk size.
```

Comment:
Well, I had referred this link: https://kubernetes.io/docs/user-guide/persistent-volumes/

> Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk, and Cinder volumes support deletion.